[2023-05-01T17:44:07.385+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:44:06.224539+00:00 [queued]>
[2023-05-01T17:44:07.398+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:44:06.224539+00:00 [queued]>
[2023-05-01T17:44:07.399+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T17:44:07.399+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-05-01T17:44:07.400+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T17:44:07.431+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-05-01 17:44:06.224539+00:00
[2023-05-01T17:44:07.443+0000] {standard_task_runner.py:55} INFO - Started process 18586 to run task
[2023-05-01T17:44:07.448+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-05-01T17:44:06.224539+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/processing_user.py', '--cfg-path', '/tmp/tmpevd9_90r']
[2023-05-01T17:44:07.452+0000] {standard_task_runner.py:83} INFO - Job 55: Subtask is_api_available
[2023-05-01T17:44:07.555+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-05-01T17:44:06.224539+00:00 [running]> on host 9bd2cedcb1a8
[2023-05-01T17:44:07.704+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-05-01T17:44:06.224539+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-01T17:44:06.224539+00:00
[2023-05-01T17:44:07.706+0000] {http.py:120} INFO - Poking: api/
[2023-05-01T17:44:07.724+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-05-01T17:44:07.725+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-05-01T17:44:08.414+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-05-01T17:44:08.425+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230501T174406, start_date=20230501T174407, end_date=20230501T174408
[2023-05-01T17:44:08.466+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-05-01T17:44:08.488+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
